wikimedia/wikipedia-ios
# Talk pages - Prevent user from replying with empty comment

**Phabricator:** N/A

### Notes
Matching the new topic view, prevent the user from replying to a comment with an empty message. The publish button should only enable if the user enters a non-empty message.

### Test Steps
1. Open talk pages and tap reply on a comment. Confirm the publish button state is disabled.
2. Confirm the publish button remains disabled when entering only any empty characters (spaces or new lines).
3. Confirm that entering any non-empty characters changes the publish button to enabled.
